src/payment/strategies/afterpay/afterpay-payment-strategy.ts

```diff
@@ -106,23 +106,26 @@ export default class AfterpayPaymentStrategy implements PaymentStrategy {
             throw new NotInitializedError(NotInitializedErrorType.PaymentNotInitialized);
         }
 
-        this._afterpaySdk.initialize({ countryCode: this._mapCountryToISO2(countryName)});
+        this._afterpaySdk.initialize({ countryCode: countryName });
```

Comment:
change the parameter name to **countryCode**

```suggestion
        this._afterpaySdk.initialize({ countryCode });
```